PAA.html

```diff
@@ -190,7 +190,16 @@ <h2>PAA</h2>
                 
 <p class="rvps2"><span class="rvts20">Polar Alignment Assistance (PAA)</span></p>
 <p class="rvps2"><span class="rvts6"><br/></span></p>
-<p class="rvps2"><span class="rvts6">If your mount supports Polar Alignment, you can use the Polar Alignment Assistant (PAA) to improve your alignment.&nbsp;</span></p>
+<p class="rvps2"><span class="rvts6">If your mount supports Polar Alignment, you can use the Polar Alignment Assistant (PAA) to improve your alignment.</span></p>
+<p class="rvps2"><span class="rvts6"><br/></span></p>
+<p class="rvps5"><span class="rvts6">The key is that you need a clear view of some stars, but they don't specifically need to be Polaris. The tool will analyze the star's movement pattern to determine the alignment corrections needed.</span></p>
+<p class="rvps5"><span class="rvts6">However, for best results:</span></p>
+<ul style="text-indent: 0px; padding: 0; margin: 0 0 0 24px; list-style-position: outside; list-style-type: disc;">
+ <li class="rvps5 noindent"><span class="rvts6">Choose a star that's relatively high in the sky (around 30-60 degrees altitude)</span></li>
+ <li class="rvps5 noindent"><span class="rvts6">Make sure your mount is roughly aligned (within several degrees)</span></li>
+ <li class="rvps5 noindent"><span class="rvts6">Ensure your mount is level</span></li>
+ <li class="rvps5 noindent"><span class="rvts6">Use a star that isn't too close to the meridian&nbsp;</span></li>
```

Comment:
Use a star that isn't too close to the meridian&nbsp;

Use a star that isn't too close to the meridian (To prevent a meridian flip)